src/app/screens/BookDetail/components/BookDetail/styles.js

```diff
@@ -0,0 +1,71 @@
+import { StyleSheet } from 'react-native';
+import colors from '../../../../../constants/colors';
+
+export default StyleSheet.create({
+  cover: {
+    height: 105,
+    width: 69,
+    maxHeight: '100%',
+    maxWidth: '100%'
+  },
+  container: {
+    flex: 1,
+    flexDirection: 'column',
+    height: '100%',
+    width: '100%',
```

Comment:
extract this 2 into another style that will be full size

Reply:
ok, done.